lib/dropwizard/databases/src/main/java/com/wikia/dropwizard/databases/DatabaseInstaller.java

```
@@ -0,0 +1,50 @@
+package com.wikia.dropwizard.databases;
+
+import com.google.inject.Binder;
+import io.dropwizard.setup.Environment;
+import ru.vyarus.dropwizard.guice.module.installer.FeatureInstaller;
+import ru.vyarus.dropwizard.guice.module.installer.install.InstanceInstaller;
+import ru.vyarus.dropwizard.guice.module.installer.install.binding.BindingInstaller;
+import ru.vyarus.dropwizard.guice.module.installer.util.FeatureUtils;
+
+public class DatabaseInstaller implements FeatureInstaller<DatabaseProvider>,
+                                          InstanceInstaller<DatabaseProvider>,
+                                          BindingInstaller {
+
+  @Override
+  public void install(Environment environment, DatabaseProvider instance) {
+    instance.getDatabases().getConfigurations().forEach((name, dbSource) -> {
+      dbSource.ensureDataSources(environment.metrics(), name);
+
+      try {
+        DatabaseHealthCheck masterCheck = new DatabaseHealthCheck(
+            dbSource.getMasterSource(),
+            dbSource.getMasterFactory().getValidationQuery());
+        environment.healthChecks().register(name+"-db-master", masterCheck);
+      } catch (NullPointerException npe) {}
```

I think you should log the npe instead of silently ignoring it.